notebooks/Shuvomoy - Getting started with JuMP.ipynb

```diff
@@ -93,7 +93,9 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "INFO: Nothing to be done\n"
+      "INFO: Nothing to be done\n",
+      "INFO: METADATA is out-of-date — you may not have the latest version of JuMP\n",
```

Comment:
Yup, will do it soon.

On Mon, Jan 25, 2016 at 6:26 PM, Miles Lubin <notifications@github.com>
wrote:

> In notebooks/Shuvomoy - Getting started with JuMP.ipynb
> <https://github.com/JuliaOpt/juliaopt-notebooks/pull/17#discussion_r50773843>
> :
>
> > @@ -93,7 +93,9 @@
> >       "name": "stderr",
> >       "output_type": "stream",
> >       "text": [
> > -      "INFO: Nothing to be done\n"
> > +      "INFO: Nothing to be done\n",
> > +      "INFO: METADATA is out-of-date — you may not have the latest version of JuMP\n",
>
> May want to rerun and clear these "out-of-date" messages
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaOpt/juliaopt-notebooks/pull/17/files#r50773843>.
>



-- 
Shuvomoy Das Gupta
Ph.D. Student
Systems Control Group
Department of Electrical & Computer Engineering
University of Toronto
http://scg.utoronto.ca/~shuvomoy.dasgupta/